### PR TITLE
Migration files are editable

### PR DESCRIPTION
**Pull request by perryr16**
## What's this PR do?
* Creates a master migration file
## Where should the reviewer start? 
* migrations/versions
## How should this be manually tested?
* pytest
* check heroku
## Any background context you want to provide?
* We should not be deleting migration files. Instead we should be modifying them.
## What are the relevant tickets?
* n/a

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,8 +15,5 @@ deploy:
   api_key:
     secure: qofM3BtlKhNbnirVWpHS15WPXDWH88MNEbo8AyVM+Y08AvzDUdbgbHQxtPn07Ke1K0X2ZexqNLV6TDbatvIW3nXxPm23DtSySb9DVpegpW/TD+C0ryO4emta3eaRmOIbnt+Aq8XSKnOBr6BOchW9EKJnFmCexJqiq935SarELtmU+J3LncEwZQuTQyVs8UpwBFtOryBkugFwRVhMjpHYqChsu+bgsDToMbjCZp8YD5e9GHo6j2Jqf3DJ+/z9UBKvtuF1brUlv8tOxIDnmdCDcaOLkewifSafn5GjtYGjJNVlqOL5mQsLEYMmWrRT7kxiVWbW08hsW1XT3ZGmhHg3nDOsYFqOGh0ezV7wD5D4/4FfWr+R26E8yXkTZXUJNsgihs6IjvDVreeRsgTe8SpfcEHViE75qddernKIPQywXPLqkUzo4aOug9Eb+2rYgKRQMCj91WDqNTiuWAAuBxrhgpyY5vgI31bZv21dhqYALMH9KrS4ZztN8qfM18wCHhXju5qOqT1mocgZoVyK5piBKcwijPQHhIn0vpjXY70GsvWO8T3gXBSLv1ucpIfnHjLW46LU0oqimkWQnpMYtHtfDAituLiCQsC8VRzZHyrzRAe7bE5DNeByPRjmALJc2HmLT2rp8TR6B9KG3mo9dpx5Fbtm21sohFDaZp2nsWrefuM=
   app: etho-boost
-  run: pipenv shell
-  run: python manage.py db stamp head
-  run: python manage.py db migrate
-  run: python manage.py db upgrade
```
Comment (perryr16) [inline .travis.yml:21]: dont need any of those silly commands. if there is a file in heroku's `migrations/versions` then there is no need to stamp, init, or migrate. Just `upgrade`

File: migrations/versions/96a8d065e5ea_.py [hunk 2]
```diff
@@ -18,6 +18,38 @@
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('campaigns',
```
Comment (perryr16) [inline migrations/versions/96a8d065e5ea_.py:21]: the master migration should create all tables `campaigns, requests, contributors, and campaign_contributors`

**Review by perryr16 — COMMENTED** [review]
take that heroku

File: .travis.yml [hunk 3]
```diff
@@ -15,8 +15,6 @@ deploy:
...
+  run: python3 manage.py db downgrade
+  run: python3 manage.py db upgrade
```
Comment (AntonioJacksonII) [inline .travis.yml:19]: Nice! This is much more straightforward.

**Review by AntonioJacksonII — APPROVED** [review]
This looks great! Thanks for deploying to Heroku.

**Review by takeller — APPROVED** [review]
Nice! This will be way easier to work with! Thanks for getting this worked out. I've been feeling confused about the migrations and how alembic handles things.

**Pull request merged**